addons/mail/models/res_config_settings.py

```diff
@@ -35,6 +35,10 @@ class ResConfigSettings(models.TransientModel):
         'Twilio Account Auth Token',
         config_parameter='mail.twilio_account_token',
     )
+    link_preview_throttle = fields.Char(
+        'Link Preview Throttle value',
+        config_parameter='mail.link_preview_throttle',
+    )
```

Comment:
It does not look like something that should be displayed in settings, it's probably best to just keep it as a more technical config parameter.

Reply:
Comment still open.

Reply:
I don't understand, it's not displayed inside the template anymore. Should I also remove the field ?